bigquery_etl/shredder/bug1751979.py

```diff
@@ -0,0 +1,251 @@
+              "ffnt",
+              "ffocus",
+              "ffos",
+              "ffsb",
+              "fpas",
+              "fpsa",
+              "ftas",
+              "ftsa",
+              "newext",
+              "monline_dg",
+              "monline_3_dg",
+              "monline_4_dg",
+              "monline_7_dg"
+            ),
+            key,
+            CONCAT(prefix, "other.scrubbed")
+          ) AS key,
+          value
+        )
+      )
+    FROM
+      base
+  )
+);
+"""
+
+BUG_1751979_MAIN_V4_REPLACE_CLAUSE = """
+REPLACE (
+(
+  SELECT AS STRUCT
+    payload.* REPLACE (
+      (
+        SELECT AS STRUCT
+          payload.keyed_histograms.* REPLACE (
+            sanitize_search_counts(payload.keyed_histograms.search_counts) AS search_counts
+          )
+      ) AS keyed_histograms,
+      (
+        SELECT AS STRUCT
+          payload.processes.* REPLACE (
+            (
+              SELECT AS STRUCT
+                payload.processes.parent.* REPLACE (
+                  (
+                    SELECT AS STRUCT
+                      payload.processes.parent.keyed_scalars.* REPLACE (
+  sanitize_scalar(
+    payload.processes.parent.keyed_scalars.browser_search_content_urlbar
+  ) AS browser_search_content_urlbar,
+  sanitize_scalar(
+    payload.processes.parent.keyed_scalars.browser_search_content_urlbar_handoff
+  ) AS browser_search_content_urlbar_handoff,
+  sanitize_scalar(
+    payload.processes.parent.keyed_scalars.browser_search_content_urlbar_searchmode
+  ) AS browser_search_content_urlbar_searchmode,
+  sanitize_scalar(
+    payload.processes.parent.keyed_scalars.browser_search_content_searchbar
+  ) AS browser_search_content_searchbar,
+  sanitize_scalar(
+    payload.processes.parent.keyed_scalars.browser_search_content_about_home
+  ) AS browser_search_content_about_home,
+  sanitize_scalar(
+    payload.processes.parent.keyed_scalars.browser_search_content_about_newtab
+  ) AS browser_search_content_about_newtab,
+  sanitize_scalar(
+    payload.processes.parent.keyed_scalars.browser_search_content_contextmenu
+  ) AS browser_search_content_contextmenu,
+  sanitize_scalar(
+    payload.processes.parent.keyed_scalars.browser_search_content_webextension
+  ) AS browser_search_content_webextension,
+  sanitize_scalar(
+    payload.processes.parent.keyed_scalars.browser_search_content_system
+  ) AS browser_search_content_system,
+  sanitize_scalar(
+    payload.processes.parent.keyed_scalars.browser_search_content_tabhistory
+  ) AS browser_search_content_tabhistory,
+  sanitize_scalar(
+    payload.processes.parent.keyed_scalars.browser_search_content_reload
+  ) AS browser_search_content_reload,
+  sanitize_scalar(
+    payload.processes.parent.keyed_scalars.browser_search_content_unknown
+  ) AS browser_search_content_unknown
+                      )
+                  ) AS keyed_scalars
+                )
+            ) AS parent
+          )
+      ) AS processes
+    )
+) AS payload
```

Comment:
this is missing a closing `)`:
```suggestion
) AS payload
)
```